node-repository/src/test/java/com/yahoo/vespa/hosted/provision/maintenance/RetiredExpirerTest.java

```diff
@@ -191,25 +190,24 @@ public void config_server_reprovisioning() throws OrchestrationException {
 
         // Redeploy to retire all 3 config servers
         infraDeployer.activateAllSupportedInfraApplications(true);
-
-        // Only 1 config server is allowed to retire at any given point in time
-        List<Node> retiredNodes = tester.nodeRepository().nodes().list(() -> {}).stream()
-                .filter(node -> node.allocation().map(allocation -> allocation.membership().retired()).orElse(false))
-                .collect(Collectors.toList());
-        assertEquals(1, retiredNodes.size());
-        Node retiredNode = retiredNodes.get(0);
-        String retiredNodeHostname = retiredNode.hostname();
-
-        // Allow retiredNodeHostname to be removed
+        List<Node> retiredNodes = tester.nodeRepository()
```

Comment:
FYI: There is a `NodeList::retired` method that does all this

Reply:
Changed accordingly